Tasks/AzurePowerShellV4/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 4,
         "Minor": 0,
-        "Patch": 7
+        "Patch": 8
```

Comment:
In some tasks we are modifying the minor and in others the patch?

Reply:
I just followed the convention for each task - its non-breaking and doesn't introduce anything new so normally it would be patch, but some tasks version by sprint